EnglishFilter/sections/antiadblock.txt

```diff
@@ -4134,7 +4134,9 @@ mega-debrid.eu#%#window.Advertisement = 1;
 file-upload.com$$script[tag-content="getElementById(""adblockinfo"")"][max-length="300"]
 file-upload.com#%#AG_abortInlineScript(/getElementById\("adblockinfo"\)/, 'document.getElementById');
 ! https://github.com/AdguardTeam/AdguardFilters/issues/58160
-tutorials-technology.info,unique-tutorials.info,getintopc.com,solvetube.site,tech-platform.info#%#//scriptlet("abort-current-inline-script", "jQuery", "ai_get_cookie")
+! https://github.com/AdguardTeam/AdguardFilters/issues/61814
+tutorials-technology.info,unique-tutorials.info,getintopc.com,solvetube.site,tech-platform.info#%#//scriptlet("abort-current-inline-script", "jQuery", "ai_")
+tutorials-technology.info,unique-tutorials.info,getintopc.com,solvetube.site,tech-platform.info#@%#//scriptlet("abort-current-inline-script", "jQuery", "ai_get_cookie")
```

Comment:
I guess I should explain it more precisely.
I meant that we can just change the current rule (`#%#//scriptlet("abort-current-inline-script", "jQuery", "ai_get_cookie")` to something like:
`#%#//scriptlet("abort-current-inline-script", "jQuery", "ai_")`

Exception rule was only to user filter otherwise "new" rule wasn't working, so could you please remove this rule:
```
tutorials-technology.info,unique-tutorials.info,getintopc.com,solvetube.site,tech-platform.info#@%#//scriptlet("abort-current-inline-script", "jQuery", "ai_get_cookie")
```

Reply:
my bad